Protein/README.md

```diff
@@ -1,30 +1,38 @@
-# Protein specification v. 0.4 
+# Protein Specification 
 
-**Profile** 
+## Type
 
-Thing > BioChemEntity(Protein profile)
+Thing > BioChemEntity > [Protein](https://bioschemas.org/Protein)
 
-**Bioschemas specification describing a Protein (BioChemEntity profile) in Life Sciences** 
+## Profile
+The [Protein Profile](https://bioschemas.org/profiles/Protein) provides details for providing markup for a protein resource.
+
+### Links to other relevant profiles
+
+Protein records will be included in a Dataset which will belong to a DataCatalog. A Protein can also be related to Protein Annotation or Protein Structure
+
+- [DataCatalog](https://bioschemas.org/profiles/DataCatalog/)
+- [Dataset](https://bioschemas.org/profiles/Dataset/)
+- [ProteinAnnotation](https://bioschemas.org/profiles/ProteinAnnotation/)
+- [ProteinStructure](https://bioschemas.org/profiles/ProteinStructure/)
+
+### Example Markup
+
+This repository contains example markup for the Protein profile generated by the [Proteins group](https://bioschemas.org/groups/Proteins/). Examples are purely illustrative and may use fictional data. 
+
+Examples should be stored in a subfolder indicating which version of the profile it is valid for. Please check to ensure the example you are reading matches the version of the profile you are interested in. 
+
+### Data Record
```

Comment:
I've renamed the heading and made it the next level down (a982292daa02551c4b07c646f09aa91032629f21)

Let me know if that works for you, otherwise it might be best just to ditch the subsection entirely.